frontends/web/src/routes/account/send/send.tsx

```diff
@@ -477,8 +483,8 @@ class Send extends Component<Props, State> {
         let updateState = {
             recipientAddress: address,
             sendAll: false,
-            fiatAmount: '',
-        };
+            fiatAmount: ''
+        } as Pick<State, keyof State>;
```

Comment:
next level :sunglasses:

Reply:
I don't like it. :)

Reply:
It could be re-written with fancy ES6 spread operator to conditionally add amount... but keep as is now.